NEWS.md

```diff
@@ -0,0 +1,25 @@
+ROhdsiWebApi 0.1.0
+=====================
+Note: this version has breaking changes because design changes and naming conventions.
+
+1. Generic design pattern: Based on discussions here https://github.com/OHDSI/ROhdsiWebApi/issues/37
+2. Consistent naming conventions: https://github.com/OHDSI/ROhdsiWebApi/issues/86 Use of WebApi instead of Atlas https://github.com/OHDSI/ROhdsiWebApi/issues/21 and case https://github.com/OHDSI/ROhdsiWebApi/issues/22
+3. Created Vignette
+4. New functions: 
+	Incidence rate https://github.com/OHDSI/ROhdsiWebApi/issues/59
+	Patient profile https://github.com/OHDSI/ROhdsiWebApi/issues/57
+	Ability to post definitions https://github.com/OHDSI/ROhdsiWebApi/issues/42
+	Use checkmate for informative error message https://github.com/OHDSI/ROhdsiWebApi/issues/56
+	Deprecated functions not following design pattern https://github.com/OHDSI/ROhdsiWebApi/issues/66
+	Estimation specification https://github.com/OHDSI/ROhdsiWebApi/issues/69
+	Retrieve a list of definition metadata https://github.com/OHDSI/ROhdsiWebApi/issues/40
+	Delete definitions https://github.com/OHDSI/ROhdsiWebApi/issues/38
+5. New ROxygen support https://github.com/OHDSI/ROhdsiWebApi/issues/44
+	
+	
+ROhdsiWebApi 0.0.3
```

Comment:
No point in adding old releases if we're not going to document the changes in them. Please remove.

Reply:
Should we consider v0.1.0 as the official first release? If yes, that would change the way we document News.Rd and we can start over.

i.e. there would be no 'changes' as this is the first release and we can start clean

Reply:
> Should we consider v0.1.0 as the official first release? If yes, that would change the way we document News.Rd and we can start over.
> 
> i.e. there would be no 'changes' as this is the first release and we can start clean